platformio.ini

```diff
@@ -70,15 +105,50 @@ HAS_DGUS_LCD            = src_filter=+<src/lcd/extui/lib/dgus>
-CASE_LIGHT_ENABLE       = src_filter=+<src/feature/caselight> +<src/gcode/feature/caselight>
+BACKLASH_COMPENSATION   = src_filter=+<src/feature/backlash.cpp>
+BARICUDA                = src_filter=+<src/feature/baricuda.cpp>
 BINARY_FILE_TRANSFER    = src_filter=+<src/feature/binary_stream.cpp> +<src/libs/heatshrink>
-(ESP3D_)?WIFISUPPORT = AsyncTCP, ESP Async WebServer
+BLTOUCH                 = src_filter=+<src/feature/bltouch.cpp>
+CANCEL_OBJECTS          = src_filter=+<src/feature/cancel_object.cpp>
+CASE_LIGHT_ENABLE       = src_filter=+<src/feature/caselight> +<src/gcode/feature/caselight>
+EXTERNAL_CLOSED_LOOP_CONTROLLER = src_filter=+<src/feature/closedloop.cpp>
+USE_CONTROLLER_FAN      = src_filter=+<src/feature/controllerfan.cpp>
+DAC_STEPPER_CURRENT     = src_filter=+<src/feature/dac>
+DIRECT_STEPPING         = src_filter=+<src/feature/direct_stepping.cpp> +<src/gcode/motion/G6.cpp>
+EMERGENCY_PARSER        = src_filter=+<src/feature/e_parser.cpp>
+I2C_POSITION_ENCODERS   = src_filter=+<src/feature/encoder_i2c.cpp>
+HAS_FANMUX              = src_filter=+<src/feature/fanmux.cpp>
+FILAMENT_WIDTH_SENSOR   = src_filter=+<src/feature/filwidth.cpp>
+FWRETRACT               = src_filter=+<src/feature/fwretract.cpp>
+HOST_ACTION_COMMANDS    = src_filter=+<src/feature/host_actions.cpp>
+HOTEND_IDLE_TIMEOUT     = src_filter=+<src/feature/hotend_idle.cpp>
+JOYSTICK                = src_filter=+<src/feature/joystick.cpp>
+BLINKM                  = src_filter=+<src/feature/leds/blinkm.cpp>
+HAS_COLOR_LEDS          = src_filter=+<src/feature/leds/leds.cpp>
+PCA9533                 = src_filter=+<src/feature/leds/pca9533.cpp>
+PCA9632                 = src_filter=+<src/feature/leds/pca9632.cpp>
+PRINTER_EVENT_LEDS      = src_filter=+<src/feature/leds/printer_event_leds.cpp>
+TEMP_STAT_LEDS          = src_filter=+<src/feature/leds/tempstat.cpp>
+MAX7219_DEBUG           = src_filter=+<src/feature/max7219.cpp>
+MIXING_EXTRUDER         = src_filter=+<src/feature/mixing.cpp>
+ADVANCED_PAUSE_FEATURE  = src_filter=+<src/feature/pause.cpp>
+AUTO_POWER_CONTROL      = src_filter=+<src/feature/power.cpp>
+HAS_POWER_MONITOR       = src_filter=+<src/feature/power_monitor.cpp>
+POWER_LOSS_RECOVERY     = src_filter=+<src/feature/powerloss.cpp>
+PROBE_TEMP_COMPENSATION = src_filter=+<src/feature/probe_temp_comp.cpp>
+HAS_FILAMENT_SENSOR     = src_filter=+<src/feature/runout.cpp>
+MK2_MULTIPLEXER         = src_filter=+<src/feature/snmm.cpp>
+EXT_SOLENOID|MANUAL_SOLENOID_CONTROL = src_filter=+<src/feature/solenoid.cpp>
+HAS_CUTTER              = src_filter=+<src/feature/spindle_laser.cpp> +<src/gcode/control/M3-M5.cpp>
+EXPERIMENTAL_I2CBUS     = src_filter=+<src/feature/twibus.cpp>
+Z_STEPPER_AUTO_ALIGN    = src_filter=+<src/feature/z_stepper_align.cpp>
+NEED_HEX_PRINT          = src_filter=+<src/libs/hex_print.cpp>
+NEED_LSF                = src_filter=+<src/libs/least_squares_fit.cpp>
+(ESP3D_)?WIFISUPPORT    = AsyncTCP, ESP Async WebServer
```

Comment:
The kid is growing!

Reply:
Currently in one of those awkward phases.